core/sdk-core/src/main/java/software/amazon/awssdk/core/async/listener/PublisherListener.java

```diff
@@ -72,4 +76,17 @@ static void invoke(Runnable runnable, String callbackName) {
             }
         }
     }
+
+    @SdkInternalApi
+    final class NoOpPublisherListener implements PublisherListener<Long> {
```

Comment:
`static`?

Reply:
checkstyle fails flagging it as redundant, reverting